test/integration-test.ts

```diff
@@ -136,10 +136,25 @@ describe('Xpring JS Integration Tests', function(): void {
     assert.equal(doesExist, true)
   })
 
-  it('Check if Account Exists - rippled', async function(): Promise<void> {
+  it('Check if Account Exists - true - rippled', async function(): Promise<
+    void
+  > {
     this.timeout(timeoutMs)
 
     const doesExist = await xpringClient.accountExists(recipientAddress)
+
     assert.equal(doesExist, true)
   })
+
+  // Omitting false-case tests for LegacyNode and LegacyWeb
```

Comment:
Can we be slightly more descriptive here? 

// Tests for account existence on the legacy shim are intentionally omitted as the legacy shim does have the required granularity of information.

Reply:
updated